python/llm/src/bigdl/llm/transformers/models/llama.py

```diff
@@ -507,7 +506,17 @@ def llama_attention_selective_batching_forward_4_31(
 
 
 def check_flash_attention_available(query):
```

Comment:
change to `use_flash_attention`?

Reply:
Done, have changed to `use_flash_attention`.